StortSpelProjekt/Engine/src/Renderer/HLSL/ForwardPixel.hlsl

```diff
@@ -92,7 +92,7 @@ PS_OUTPUT PS_main(VS_OUT input)
 			baseReflectivity);
 	}
 	
-	float3 ambient = float3(0.004f, 0.004f, 0.004f) * albedo;
+	float3 ambient = float3(0.05f, 0.05f, 0.05f) * albedo;
 	finalColor += ambient;
```

Comment:
Change this back to 0.004f. Fredrik told me to say that he changed this temporary and forgot to change them back.